src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -95,6 +95,12 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
             if (informed.hasRouteId()) {
                 routeId = informed.getRouteId();
             }
+
+            String direction = null;
+            if (informed.hasTrip() && informed.getTrip().hasDirectionId()) {
+                direction = String.valueOf(informed.getTrip().getDirectionId());
```

Comment:
Actually the return value of `getDirectionId()` is int, as it is specified as 
```
// The direction_id from the GTFS feed trips.txt file, indicating the direction of travel for this trip.
optional uint32 direction_id = 6;
```
In the gtfs-realtime.proto